renderlib/Logging.cpp

```diff
@@ -68,3 +76,38 @@ Logging::Enable(bool enabled)
 {
   boost::log::core::get()->set_logging_enabled(enabled);
 }
+
+boost::filesystem::path
+getLogPath()
+{
+#if defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__)
+  const char* rootdir = getenv("LOCALAPPDATA");
+  return boost::filesystem::path(rootdir) / "AllenInstitute" / "agave";
+#elif __APPLE__
+  const char* rootdir = getenv("HOME");

```

Comment:
TBH, I wanted to avoid working on this for a week implementing the most robust possible cross-platform "get home directory" function.  This one should work for most typical users.  It was sourced from stackoverflow (I'll have to re-find the link).  Granted, users can change environment variables but it would be a fairly nonstandard thing.   What do you recommend?  Should I be using getpwuid without bothering to check the HOME var?

Reply:
https://stackoverflow.com/questions/3020187/getting-home-directory-in-mac-os-x-using-c-language

Reply:
also this one... https://stackoverflow.com/a/3733955 .. i could probably write a getUserHomeDirectory function that handles all three but on Windows, the LOCALAPPDATA gets me a little further along

Reply:
I looked at options on this and it just seemed redundant unless you wanted people to be able to fiddle with $HOME